language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/AbstractImplementMethodCodeAction.java

```diff
@@ -153,9 +154,33 @@ public AbstractImplementMethodCodeAction(List<CodeActionNodeType> nodeTypes) {
         ImportsAcceptor importsAcceptor = new ImportsAcceptor(context);
         String typeName = FunctionGenerator.processModuleIDsInText(importsAcceptor, unimplMethod.signature(), context);
         List<TextEdit> edits = new ArrayList<>(importsAcceptor.getNewImportTextEdits());
-        String editText = offsetStr + typeName + " {" + LINE_SEPARATOR + offsetStr + "}" + LINE_SEPARATOR;
+
+        String returnStmt = "";
+        if (unimplMethod.typeDescriptor().returnTypeDescriptor().isPresent()) {
+            TypeSymbol returnTypeSymbol = unimplMethod.typeDescriptor().returnTypeDescriptor().get();
+            Optional<String> returnType = FunctionGenerator.getReturnTypeAsString(context, returnTypeSymbol);
```

Comment:
What if we simply do 
```
returnStmt = "return " + CommonUtil.getDefaultValueForType(returnTypeSymbol) +
                        CommonKeys.SEMI_COLON_SYMBOL_KEY;
```
without invoking `returnTypeAsString()` here? Only use case of invoking that is to avoid compilation error type symbols. WDYT?

Reply:
+1